binding.gyp

```diff
@@ -38,6 +38,23 @@
 					} ]
 				},
 				{
+					"target_name": "copyapis",
+					"type": "none",
+					"conditions": [ [ "'<!(echo $BUILD_NPM_PACKAGE)'=='y'", {
+						"actions": [ {
+							"action_name": "copyapis",
+							"message": "Copying JS APIs",
+							"outputs": [ "./uart.js" ],
```

Comment:
Hmmm ... maybe this needs a rethink. What if soletta is built without uart support? The fact is that this is just an artifact to make sure the target has some inputs and outputs. Maybe this target must also be generated by configure-bindings.js. Good catch, @edersondisouza!